src/shared/tasks/sentry/tasksentry.cpp

```diff
@@ -76,12 +76,20 @@ void TaskSentry::run() {
 
 void TaskSentry::sendRequest() {
   NetworkRequest* request = new NetworkRequest(this, 200);
+  auto settings = SettingsHolder::instance();
+  auto endpoint = settings->sentryEndpoint();
+  auto dsn = settings->sentryDSN();
+  if (endpoint.isNull() || dsn.isNull()) {
+    // We could have not even been initializing sentry without that?!
```

Comment:
Should we rephrase a comment like this to something a little more specific? maybe like:
```suggestion
    // We should not be able to initialize Sentry without an endpoint and DSN
```